<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibraryInfo</class>
 <widget class="QDialog" name="LibraryInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1205</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="libStatus">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>351</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Comments</string>
   </property>
   <widget class="QPushButton" name="pushButton_PostComment">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Comment">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>129</y>
      <width>256</width>
      <height>191</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Comment</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>165</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ISBN   </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_ISBN"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>165</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Rank   </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Rank"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_AddorModify">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Add/Modify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Search">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Search ISBN</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>361</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Recommendation</string>
   </property>
   <widget class="QPushButton" name="pushButton_Rank">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Top Ranking Books</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Person">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Personal Recommend</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_Rec">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>311</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="welcomeTag">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>481</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome!</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>50</y>
     <width>351</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Book Information</string>
   </property>
   <widget class="QTableView" name="tableView_BookInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>170</y>
     <width>351</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Comments and Ratings</string>
   </property>
   <widget class="QTableView" name="tableView_Comm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_commentCount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
